pkg/minishift/docker/image/image.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"fmt"
+	"github.com/minishift/minishift/pkg/util/os"
+	"io"
+	"os/exec"
+)
+
+type ImageMissStrategy int
+
+const (
+	SKIP ImageMissStrategy = iota
+	PULL
+	// TODO Implement a retry strategy as well (HF)
+	//RETRY
+)
+
+type ImageCacheConfig struct {
+	HostCacheDir string
+	CachedImages []string
+	Out          io.Writer
+	MissStrategy ImageMissStrategy
```

Comment:
I guess you mean _ImageMissStrategy_. Fair enough. It is not a cache miss though.